src/plugins/query_enhancements/common/utils.ts

```diff
@@ -133,7 +133,20 @@ export const handleDataFrameError = (response: any) => {
   }
 };
 
-export const fetchDataFrame = (context: FetchDataFrameContext, query: Query, df: IDataFrame) => {
+export const fetch = (context: EnhancedFetchContext, query: Query, aggConfig?: QueryAggConfig) => {
```

Comment:
Add some comments here to these functions. They seem to be specific to PPL and SQL. I.e SQL plugin related features and not the default DSL based fetch